src/common/options/mds-client.yaml.in

```diff
@@ -546,3 +546,14 @@ options:
   min: 0
   flags:
   - runtime
+- name: client_disable_collect_and_send_global_metrics
+  type: bool
+  level: advanced
+  desc: to disable collocting and sending the global metrics to MDS
+  default: false
+  tags:
+  - client
+  services:
+  - mds_client
+  flags:
+  - runtime
```

Comment:
Here too, I'd like to suggest you to avoid the double negation.
Please change the option name to something like: `client_collect_and_send_global_metrics` and then decide on the default value.

Reply:
Sure, will fix it. Thanks @mchangir